feat: Populate compute proto with extended_operations annotations

vam-google opened the pull request:
Also update WORKSPACE file to match the latest from googleapis/googleapis repo

vam-google committed:
feat: Populate compute proto with extended_operations annotations

Also update WORKSPACE file to match the latest from googleapis/googleapis repo

bshaffer commented on google/cloud/compute/v1/compute_small.proto:
```diff
@@ -686,6 +701,7 @@ service RegionOperations {
       get: "/compute/v1/projects/{project}/regions/{region}/operations/{operation}"
     };
     option (google.api.method_signature) = "project,region,operation";
+    option (operation_polling_method) = true;
```
And shouldn't there also be an `operation_cancel_method` and `operation_delete_method` annotation? How else will we know that the `Delete` method is used for this purpose?

viacheslav-rostovtsev replied:
- the `operation_polling_method` it's not in the design doc, but it's in the new annotations: https://source.corp.google.com/piper///depot/google3/third_party/googleapis/google/cloud/extended_operations.proto;rcl=392508615;l=120
- there seems to be no need for `operation_cancel_method` since AFAIK Gapic LRO wrappers do not provide cancel of delete functionality, only 'poll until completion'

vchudnov-g replied:
Thanks for the clarification! I suggest we do mention `operation_polling_method` in the design doc, and also point from there to the checked-in `extended_operations.proto`

viacheslav-rostovtsev commented:
You have `option (operation_service)` on `rpc wait` methods. Are you sure you need that?

viacheslav-rostovtsev reviewed (changes requested):
You have `option (operation_service)` on `rpc wait` methods in the Operations services. Are you sure you need that?

vam-google commented:
@viacheslav-rostovtsev Yes, it should not be there. This PR was generated before this thing got clarified. I haven't gotten to pushing this PR yet, as I need to figure out the common-protos in each langauge first. Thanks for pointing this out, it will be regenerated without the annotation applied to wait.

vam-google committed:
Merge remote-tracking branch 'upstream/master'

vam-google committed:
regenerate with the newest disco-to-proto3-converter

add full annotation name (with package) to extended_operations
do not apply polling_operation_method to Wait() methods of the pollin service

vam-google commented:
@viacheslav-rostovtsev unnecessary annotation for Wait method was remoed. PTAL
